packages/edit-site/src/components/sidebar-navigation-screen-templates-browse/index.js

```diff
@@ -1,8 +1,8 @@
+import { __experimentalUseNavigator as useNavigator } from '@wordpress/components';
```

Comment:
This should be returned.

Reply:
Hmm.  Yeah.  This seems unrelated to this issue tho.  Want to file a separate issue for this?